src/generator/xss_report.py

```
@@ -0,0 +1,38 @@
+    prompt = render(
+        "xss_intel",
+        frameworks=frameworks,
+        libraries=libraries,
+        keywords=keywords,
+        date=date.today().strftime("%Y-%m-%d"),
+    )
+
+    logger.info("claude CLI (WebSearch) 呼び出し開始 [XSS Intel]")
+    logger.debug("対象フレームワーク: %s / ライブラリ: %s", frameworks, libraries)
+
+    result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)